server/build/docker-preview/config_docker.json

```diff
@@ -0,0 +1,424 @@
+{
+    "ServiceSettings": {
+        "SiteURL": "",
+        "WebsocketURL": "",
+        "LicenseFileLocation": "",
+        "ListenAddress": ":8065",
+        "ConnectionSecurity": "",
+        "TLSCertFile": "",
+        "TLSKeyFile": "",
+        "UseLetsEncrypt": false,
+        "LetsEncryptCertificateCacheFile": "./config/letsencrypt.cache",
+        "Forward80To443": false,
+        "ReadTimeout": 300,
+        "WriteTimeout": 300,
+        "MaximumLoginAttempts": 10,
+        "GoroutineHealthThreshold": -1,
+        "GoogleDeveloperKey": "",
+        "EnableOAuthServiceProvider": false,
+        "EnableIncomingWebhooks": true,
+        "EnableOutgoingWebhooks": true,
+        "EnableCommands": true,
+        "EnableOnlyAdminIntegrations": true,
+        "EnablePostUsernameOverride": false,
+        "EnablePostIconOverride": false,
+        "EnableAPIv3": false,
+        "EnableLinkPreviews": false,
+        "EnableTesting": false,
+        "EnableDeveloper": false,
+        "EnableSecurityFixAlert": true,
+        "EnableInsecureOutgoingConnections": false,
+        "AllowedUntrustedInternalConnections": "",
+        "EnableMultifactorAuthentication": false,
+        "EnforceMultifactorAuthentication": false,
+        "EnableUserAccessTokens": false,
+        "AllowCorsFrom": "",
+        "CorsExposedHeaders": "",
+        "CorsAllowCredentials": false,
+        "CorsDebug": false,
+        "AllowCookiesForSubdomains": false,
+        "SessionLengthWebInDays": 30,
+        "SessionLengthMobileInDays": 30,
+        "SessionLengthSSOInDays": 30,
+        "SessionCacheInMinutes": 10,
+        "SessionIdleTimeoutInMinutes": 0,
+        "WebsocketSecurePort": 443,
+        "WebsocketPort": 80,
+        "WebserverMode": "gzip",
+        "EnableCustomEmoji": false,
+        "EnableEmojiPicker": true,
+        "EnableGifPicker": false,
+        "GfycatApiKey": "2_KtH_W5",
+        "GfycatApiSecret": "3wLVZPiswc3DnaiaFoLkDvB4X0IV6CpMkj4tf2inJRsBY6-FnkT08zGmppWFgeof",
```

Comment:
Irrelevant to the work performed here. 

Have we phased out Gfycat? If so, let's take this opportunity to make the necessary updates accordingly.

Reply:
Probably we do not need that yes. @M-ZubairAhmed we can just remove the configuration right ?

Reply:
I think in general, this config is outdated and could easily get outdated even if we update now. I wonder if this config should be updated regularly or set to minimum config possible (and let it use the default values if unset). I haven't able to run yet but I'm thinking something like below only. Thoughts @isacikgoz?
```json
{
    "ServiceSettings": {
        "EnableOAuthServiceProvider": true,
        "EnableDeveloper": false,
        "EnableGifPicker": true
    },
    "SqlSettings": {
        "DriverName": "postgres",
        "DataSource": "postgres://mmuser:mostest@localhost/mattermost_test?sslmode=disable\u0026connect_timeout=10\u0026binary_parameters=yes",
        "AtRestEncryptKey": ""
    },
    "FileSettings": {
        "DriverName": "local",
        "Directory": "/mm/mattermost-data",
        "PublicLinkSalt": ""
    },
    "EmailSettings": {
        "SMTPServer": "",
        "SMTPPort": "",
        "PushNotificationContents": "generic"
    },
    "ElasticsearchSettings": {
        "ConnectionUrl": ""
    },
    "PluginSettings": {
        "EnableUploads": true,
        "PluginStates": {}
    }
}
```

Reply:
@toninis yes Gfycat have been completely phased out so we can safely remove these keys